### PR TITLE
feat: tansform jsx to html

### PR DESCRIPTION
**Pull request by chenjun1011**
pre tansform jsx to html for better ssr performance.

input 

```jsx
<div className="container" style={style} onClick={onClick}>
  <div>a {props.index}</div>
</div>
```

output

```js
[{
  __html: "<div class=\\"container\\""
}, {
  __attrs: {
    style: style,
    onClick: onClick
  }
}, {
  __html: ">"
}, [{
  __html: "<div>"
}, "a ", props.index, {
  __html: "</div>"
}], {
  __html: "</div>"
}]
```

benchmark (use inline style, about 1000 nodes)

```bash
mean:0.883ms
React#renderToString x 1,132 ops/sec ±2.17% (84 runs sampled)
mean:0.315ms
Rax#renderToString x 3,173 ops/sec ±1.64% (86 runs sampled)
```

benchmark (use className, about 1000 nodes))

```bash
mean:0.626ms
React#renderToString x 1,598 ops/sec ±3.11% (82 runs sampled)
mean:0.099ms
Rax#renderToString x 10,101 ops/sec ±1.85% (83 runs sampled)
mean:0.075ms
Marko#renderToString x 13,332 ops/sec ±2.45% (85 runs sampled)
mean:0.090ms
Xtpl#renderFile x 11,065 ops/sec ±5.92% (67 runs sampled)
```

### CODE REVIEW

**Comment by wssgcg1213** [comment]
再测一下 JSXSpreadAttribute 试试: `<div {...props} />`

**Comment by chenjun1011** [comment]
after merge sibling string

benchmark (use className, about 1000 nodes)

```bash
mean:0.634ms
React#renderToString x 1,578 ops/sec ±5.26% (82 runs sampled)
mean:0.070ms
Rax#renderToString x 14,249 ops/sec ±1.54% (83 runs sampled)
mean:0.187ms
Inferno#renderToString x 5,357 ops/sec ±1.56% (83 runs sampled)
mean:0.073ms
Marko#renderToString x 13,758 ops/sec ±1.81% (84 runs sampled)
mean:0.095ms
Xtpl#renderFile x 10,551 ops/sec ±9.05% (66 runs sampled)
```

File: packages/babel-plugin-transform-jsx-to-html/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    style: style,
+    onClick: onClick
+  }
+}, {
+  __html: ">"
+}, [{
+  __html: "<div>"
+}, "a ", props.index, {
+  __html: "</div>"
+}], {
```
Comment (yuanyan) [inline packages/babel-plugin-transform-jsx-to-html/README.md]: README 可以更新下

File: packages/babel-plugin-transform-jsx-to-html/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "babel-plugin-transform-jsx-to-html",
+  "version": "0.1.0",
+  "description": "Transform JSX to Html.",
```
Comment (yuanyan) [inline packages/babel-plugin-transform-jsx-to-html/package.json]: Html -> HTML

File: packages/babel-plugin-transform-jsx-to-html/src/__tests__/index.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+<View>
+  <div>a</div>
+  <div>b</div>
+</View>
+    `)).toBe(`createElement(View, null, [{
+  __html: "<div>a</div>"
+}], [{
+  __html: "<div>b</div>"
+}]);`);
+  });
```
Comment (yuanyan) [inline packages/babel-plugin-transform-jsx-to-html/src/__tests__/index.js]: [{
  __html: "<div>a</div>"
}], [{
  __html: "<div>b</div>"
}]   优化为   [{
  __html: "<div>a</div>"
}, {
  __html: "<div>b</div>"
}] 是不是更好？

**Pull request merged**